release/_releases_json.sh

```diff
@@ -90,13 +90,15 @@ function _process_new_product {
 			end
 		)" "${releases_json}" > temp_file.json && mv temp_file.json "${releases_json}"
 
-	_process_product_version "${LIFERAY_RELEASE_PRODUCT_NAME}" "${_PRODUCT_VERSION}"
+	local latest_product_version="$(_get_latest_product_version "${LIFERAY_RELEASE_PRODUCT_NAME}")"
+
+	_process_product_version "${LIFERAY_RELEASE_PRODUCT_NAME}" "${_PRODUCT_VERSION}" "${latest_product_version}"
```

Comment:
@davisaints, can you please invoke the function `_get_latest_product_version` as a parameter of the function `_process_product_version` instead of declare a variable used a single time?